test/WindowsTimezoneTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Calendart\Adapter\Office365\Test;
```

Comment:
Remove the `Test` namespace, it is not needed here (https://github.com/Calendart/Office365Adapter/blob/master/composer.json#L32). With PSR-4, we do not need the Test namespace :}

And you can then remove the use below, as it is, per se, the same namespace. :}